src/sgxenv.mk

```diff
@@ -3,6 +3,8 @@ INCLUDE_MAKEFILE := $(lastword $(MAKEFILE_LIST))
 CUR_DIR := $(shell dirname $(realpath $(MAIN_MAKEFILE)))
 PROJECT_DIR := $(realpath $(CUR_DIR)/../../)
 
+SHELL = /bin/bash
```

Comment:
> The Two Flavors of Variables... The first flavor of variable is a recursively expanded variable. Variables of this sort are defined by lines using ‘=’ ... To avoid all the problems and inconveniences of recursively expanded variables, there is another flavor: simply expanded variables. Simply expanded variables are defined by lines using ‘:=’...

Prefer to use simply expanded variables (defined using `:=`) over recursively expanded variables (defined using `=`).

Reply:
Done.